### DISCUSSION TITLE
Page won't update the changes. Please Help :)

### DISCUSSION THREAD

**Discussion by realmsoft0r3m** [cotes2020/jekyll-theme-chirpy#1537] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Greetings!

I need your help because the website doesn't want to update changes? 

Thank you very much

### Operations you have already tried

I have tried to unpublish site and then publish it again, but the page does not update.

**Comment by imagelessthought** [comment]
Can you provide some details?

**Reply by realmsoft0r3m** [reply]
Sure, I made a post and change the _config.yml with my own info but the updates won't show on the page when I publish it.
Here is the link https://realmsoft0r3m.github.io, the page just stays on the default and won't update the changes. You can check my repository and see the changes made in the _posts and the _config.yml but not on the page. What am I doing wrong? Thank you

**Comment by imagelessthought** [comment]
When I make changes in the _config.toml - they are not reflected until I stop the server and restart.  Did you try that?

**Reply by realmsoft0r3m** [reply]
Yes I tried to unpublish the site then set the branch to none and then update it to main again,after that the site successfully deploys but the update does not reflect.I tried it again today and now the page won't even show the theme there is just blank page with layout: home.
